lib/src/main/java/graphql/nadel/engine/NadelIncrementalResultAccumulator.kt

```diff
@@ -0,0 +1,179 @@
+package graphql.nadel.engine
+
+import graphql.GraphQLError
+import graphql.incremental.DeferPayload
+import graphql.incremental.DelayedIncrementalPartialResult
+import graphql.incremental.DelayedIncrementalPartialResultImpl
+import graphql.nadel.engine.transform.query.NadelQueryPath
+import graphql.nadel.engine.util.MutableJsonMap
+import graphql.normalized.ExecutableNormalizedField
+import graphql.normalized.ExecutableNormalizedOperation
+import graphql.normalized.incremental.NormalizedDeferredExecution
+
+class NadelIncrementalResultAccumulator(
```

Comment:
Can you add some docs on this class that briefly describes the problem that it is solving?

Reply:
Done